packages/lib-auth/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@zooniverse/auth",
+  "version": "1.0.0",
+  "main": "src/index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "axios": "~0.19.0",
+    "cookie": "~0.4.0",
+    "detect-node": "~2.0.4",
+    "yup": "~0.27.0"
+  },
+  "scripts": {
+    "docs": "jsdoc src",
+    "lint": "zoo-standard",
+    "lint:fix": "zoo-standard --fix",
+    "test": "mocha ./test/unit/playground.js",
```

Comment:
Is this cruft since `playground.js` is added to the gitignore file?

Reply:
It is, yes